middleware.go

```diff
@@ -0,0 +1,27 @@
+package main
+
+import (
+	"musical/config"
+	"net/http"
+
+	"github.com/gofiber/fiber/v2"
+	"github.com/golang-jwt/jwt"
+)
+
+func auth(c *fiber.Ctx) error {
+	tokenStr := c.Get("bearerToken")
+	if tokenStr == "" {
+		c.SendStatus(http.StatusUnauthorized)
+		return nil
```

Comment:
I don't know fiber, but shouldn't this be like:
```go
return c.SendStatus(http.StatusUnauthorized)
```

Same for other similar areas.

Reply:
oh yea, the `return nil` was filler code, this method and a couple other response methods also returns nil btw